test/demo/raiseIntent.test.ts

```diff
@@ -89,10 +90,12 @@ describe('Intent listeners and raising intents', () => {
             });
 
             describe('When the target is *not* registered to accept the raised intent', () => {
-                test('When calling raiseIntent the promise rejects with an error', async () => {
+                test('When calling raiseIntent the promise rejects with an FDC3Error', async () => {
                     const resultPromise = fdc3Remote.raiseIntent(testManagerIdentity, invalidPayload.intent, invalidPayload.context, testAppIdentity.appId);
                     // TODO: decide if this is this the error message we want when sending a targeted intent
                     await expect(resultPromise).rejects.toThrowError(/No applications available to handle this intent/);
+                    await expect(resultPromise).rejects.toHaveProperty('name', 'FDC3Error');
+                    await expect(resultPromise).rejects.toHaveProperty('code', ResolveError.NoAppsFound);
```

Comment:
For part 2, consider if it's worth making a util for this - going to be a lot of very similar asserts.

Reply:
Implemented on [this commit](https://github.com/spcfran/fdc3-service/commit/aeb68a8c83919754c10f9b25d8dcdd347ce61988), will go in v2